### PR TITLE
Protocol RSL366

### PR DESCRIPTION
**Pull request by kirichkov**
I'm resubmitting this pull request, along with updated code, because as it turns out the arctech_switch_old protocol uses different addressing of the remote devices, which makes it confusing what values to input for unit/id in the configuration file for RSL366 devices.

### CODE REVIEW

**Comment by CurlyMoo** [comment]
Can you elaborate those differences?

**Comment by kirichkov** [comment]
I've detailed them here - http://forum.pilight.org/Thread-RSL366T-Switches?pid=9164#pid9164

In short arctech uses groups 1 to 5 to identify Unit, while RSL uses 1 to 4; Arctech uses 6 to 10 to identify ID, while RSL uses 5 to 8. In RSL the position of the high pulse designates the Unit/ID. This is already properly documented in the wiki

**Comment by CurlyMoo** [comment]
Can you also remove the rsl366 from `arctech_switch_old` and update it's version number in the same PR?

Thanks for the reminder.

**Comment by kirichkov** [comment]
I saw no reference to rsl366 in arctech_switch_old.c, The only reference was the pulse length of 390, but that is present because of the Düwi Terminal Switches.
I've already amended the CMakeLists.txt as part of this PR.

File: libs/protocols/rsl366.c [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+	rsl366->parseCode=&rsl366ParseCode;
+	rsl366->createCode=&rsl366CreateCode;
+	rsl366->printHelp=&rsl366PrintHelp;
+}
+
+#ifdef MODULE
+void compatibility(struct module_t *module) {
+	module->name = "rsl366";
+	module->version = "0.8";
```
Comment (CurlyMoo) [inline libs/protocols/rsl366.c]: If it's fully working, just make the version 1.0?

File: libs/protocols/rsl366.c [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+	module->reqversion = "5.0";
+	module->reqcommit = NULL;
```
Comment (CurlyMoo) [inline libs/protocols/rsl366.c]: The required commit should be the same as in the other protocols...

**Comment by kirichkov** [comment]
Done! Should be good to go!

**Pull request merged**